docs/guides/running-clustered-experiments.md

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_position: 2
+---
+
+# Running Clustered Experiments
+
+## Overview
+
+In a standard A/B test, the analysis unit matches the randomization unit. For example, it is common to split users (the
+randomization unit)
+randomly between multiple variants and then analyze "per user" metrics such as conversion rate (defined as the
+percentage of users who convert)
+or average revenue per user.
+
+However, there are cases when it is desirable to define an analysis unit that does not match the randomization unit.
+Typically, these cases arise when
+randomizing at the same grain as the analysis would lead to undesirable interference effects. Examples include:
+
+- A business wants to measure a treatment effect on the metric _average order value_. This metric has an analysis unit
+  of an _order_; however, because users can have multiple orders, randomizing at the order level could lead to some
+  users seeing multiple variants, which is an inconsistent user experience. Also, exposure to the treatment could change
+  their behavior in a way that would persist even if they were later assigned to the control group. As a result, it's
+  better to randomize at the user level, even though the analysis metric is "per order" rather than "per user."
+- A business has other companies as customers. The business wants to measure a treatment effect on a "per user" metric.
+  However, randomizing at the user level would lead to users from the same company receiving different treatment
+  assignments, which may be problematic. For example, if the product is an internal messaging software, then an
+  effective treatment could cause users in the treatment group to send more messages to their coworkers in the control
+  group, which would also increase the engagement of the control group. As a result, it may be better to randomize at
+  the company level rather than the user level.
+
+Both of these examples illustrate use cases for running _clustered_ experiments, meaning that we randomize _groups_ of
+analysis units. Although clustered experiments can be useful
+for preventing interference issues, there is a statistical challenge because observations from the same cluster are not
+statistically independent. Treating observations from the same cluster
+as statistically independent typically leads to overconfident results (underestimating the variance).
+
+Fortunately, [Ratio Metrics](/data-management/metrics/ratio-metric)
+address this issue based on the approach described
+in [Deng et al.](https://alexdeng.github.io/public/files/kdd2018-dm.pdf) and recommended in Chapter 18 of _Trustworthy
+Online Controlled Experiments_. The approach is to recognize that a metric defined at a finer grain than the
+randomization unit can be expressed as the ratio of two metrics at the same grain as the randomization unit. To
+illustrate this,
+we will outline two examples.
+
+## Example 1: Average Order Value (AOV)
```

Comment:
Leading with this example is IMO not in the spirit of a "guide". It reads more academic: _here's this thing you took for granted, and is actually trickier than you thought; but it still falls under this approach_. The takeaway for an Eppo user is not clear: should they do anything differently?

The second example is more salient, and more clearly explains how to solve a new problem. In fact, I'd like to see more done to highlight the last sentence, and that this is the "how to" part of the guide.

Reply:
Good points. Originally, I was hoping it would read more like this: _this approach is actually less tricky than you thought; in fact, you're probably already using it even though you don't realize it_. However, I agree it is a bit academic and should emphasize the "how to" component more.

Here are some ideas to address this:
1. Reverse the order of the examples
2. Remove the AOV example entirely
3. Give more context for the AOV example without changing the order

I'll think more about it...

Reply:
I added more context to clarify some of these points. See https://github.com/Eppo-exp/eppo-docs/pull/325/commits/04e9a3c3671e342e1fb86b904b2fab540070c7b2 and https://github.com/Eppo-exp/eppo-docs/pull/325/commits/5e2f356b93e49f2fe58697ce846d7419b48c0916.

I made it more explicit that this example is a good introduction because users are likely already running clustered analyses even if they don't realize it. Hopefully this makes it clear that they don't need to do anything different with AOV for the clustered case.

Reply:
The extra context is helpful, thanks!